Competition2018/src/main/java/competition/subsystems/wrist/commands/WristDangerousOverrideCommand.java

```diff
@@ -0,0 +1,30 @@
+package competition.subsystems.wrist.commands;
+
+import com.google.inject.Inject;
+
+import competition.operator_interface.OperatorInterface;
+import competition.subsystems.wrist.WristSubsystem;
+import xbot.common.command.BaseCommand;
+
+public class WristDangerousOverrideCommand extends BaseCommand {
+    WristSubsystem wrist;
+    OperatorInterface oi;
+    
+    @Inject
+    public WristDangerousOverrideCommand(WristSubsystem wrist, OperatorInterface oi) {
+        this.wrist = wrist;
+        this.oi = oi;
+        this.requires(wrist);
+    }
+    
+    @Override
+    public void initialize() {
+        log.info("Initializing");
+    }
+
+    @Override
+    public void execute() {
+        wrist.insanelyDangerousSetPower(oi.driverGamepad.getLeftVector().y);
```

Comment:
`operatorGamepad`

Reply:
Is there a certain button I should place it on? or should I just find room for it?